.github/workflows/release.yaml

```diff
@@ -0,0 +1,34 @@
+name: release
```

Comment:
By renaming this file you've made it difficult for me to see what if anything has changed.  Can you put if back where it was please